src/fiatconnect/ReviewScreen.test.tsx

```diff
@@ -14,6 +14,32 @@ import { Currency } from 'src/utils/currencies'
 import { createMockStore, getMockStackScreenProps, sleep } from 'test/utils'
 import { mockFiatConnectQuotes } from 'test/values'
 
+const MVP_SUPPORTED_COUNTRIES: Partial<Record<FiatAccountSchema, Array<string>>> = {
```

Comment:
Where does this list come from? Is this just "countries that have some amount of support for cash out, but do not have USD as one of their currency types"? I feel like this might get out of date/unmaintained as we add support for more countries.

Reply:
This list comes directly from the 3rd AC of ACT-530. After a [discussion](https://valora-app.slack.com/archives/C02KBT0DAHJ/p1670441199274289) on slack, I interpret that AC to mean we should ensure the MVP countries receive the disclaimer if the currency is unsupported locally (which USD is). I do agree that it may get out of date with time, but it appears we are trying to verify that at least these listed countries are supported initially.

Reply:
nit: Is there a reason to test a specific list of currencies? Looking at the code, seems like we just need to test 2 cases for each schema, one with fiat type === default currency and one with !==

Reply:
Ok, I'll remove the list and just do those 2 tests per schema.